app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,8 @@
+<div>
+  <h2><%= @recipe.name%></h2>
+  <p><%= @recipe.preparation_time%>hours</p>
+  <p><%= @recipe.cooking_time%> hours</p>
+  <pre><%= @recipe.description%></pre>
```

Comment:
- [ ] Kindly add the functionality for adding ingredients and generating a shopping list here (you are generating a shopping list by default). Currently, I can't make a shopping list or add ingredients. This is a requirement.


![recipe-page](https://github.com/ezraermy/Recipe-App/assets/53124548/2e169d70-e316-44b0-86ca-66a8390aa046)


![your-recipe-page](https://github.com/ezraermy/Recipe-App/assets/53124548/e2652836-8d39-47c0-a8a6-bf9cb7c4c93b)